Source/WTF/Configurations/module.modulemap

```diff
@@ -0,0 +1,62 @@
+module wtf {
```

Comment:
Should be `[system]`

Reply:
https://github.com/WebKit/WebKit/pull/51670